### PR TITLE
Edits to About DEA user guide page

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
These edits are in response to the issues that Robbi raised with this page: https://github.com/GeoscienceAustralia/dea-knowledge-hub/pull/222#issuecomment-2123602949

I also made some additional edits to minor wording punctuation.

### CODE REVIEW

**Comment by CEKrause** [comment]
Thanks for making these changes. They look good to me.

File: docs/guides/about/intro.rst [hunk 1]
```diff
@@ -36,7 +39,7 @@ the `Commonwealth Scientific and Industrial Research Organisation`_ (CSIRO) and
...
+DEA is powered by `Open Data Cube <http://opendatacube.org/>`_ (ODC), an open-source library for the management, analysis, and processing of Geographic Information System (GIS) data — namely Earth Observation satellite data. It is a global initiative to increase the value and use of satellite data by providing users with access to free data management technologies and analysis platforms. At its core, ODC is a set of Python libraries and a PostgreSQL database that facilitate working with geospatial raster data.
 
-The ODC source code can be `found on GitHub <https://github.com/opendatacube/datacube-core>`.
+On GitHub, you will find the `ODC source code <https://github.com/opendatacube/datacube-core>`_.
```
Comment (robbibt) [inline docs/guides/about/intro.rst:44]: This new wording feels less direct and more passive - I prefer the original personally but it's a minor thing. 🙂

**Review by robbibt — APPROVED** [review]
I left a small comment, but everything else looks great to me!

File: docs/guides/about/intro.rst (hunk 1 above)
Comment (benji-glitsos-ga) [reply docs/guides/about/intro.rst:44]: I agree actually, it should be more 'active voice'. Ive reworded to:

```rst
The ODC source code can be found in the `datacube-core repository on GitHub <https://github.com/opendatacube/datacube-core>`_.
```

**Pull request merged**